app/src/main/java/ch/blinkenlights/android/vanilla/ui/CustomElements/CustomTextView.java

```diff
@@ -0,0 +1,44 @@
+package ch.blinkenlights.android.vanilla.ui.CustomElements;
+
+import android.content.Context;
+import android.graphics.Rect;
+import android.util.AttributeSet;
+
+import androidx.appcompat.widget.AppCompatTextView;
+
+
+/**
+ * The Marquee effect needs an active focus. This custom Textview is always focussed, or at least it says so. This allows the marquee to permanently play.
+ */
+public class CustomTextView extends AppCompatTextView {
+
+	public CustomTextView(Context context) {
+		super(context);
+	}
+
+	public CustomTextView(Context context, AttributeSet attrs) {
+		super(context, attrs);
+	}
+
+	public CustomTextView(Context context, AttributeSet attrs, int defStyleAttr) {
+		super(context, attrs, defStyleAttr);
+	}
+
+	@Override
+	protected void onFocusChanged(boolean focused, int direction, Rect previouslyFocusedRect) {
+		if(focused)
+			super.onFocusChanged(focused, direction, previouslyFocusedRect);
+	}
+
+	@Override
+	public void onWindowFocusChanged(boolean focused) {
+		if(focused)
+			super.onWindowFocusChanged(focused);
+	}
+
+
+	@Override
+	public boolean isFocused() {
```

Comment:
I don't think that's a good idea as this will most likely cause issues with accessibility or confuse the framework in some other way.

That said, i'm not a fan of marquee text and don't think that we should add this.

Reply:
How would you display Titles that are to long? (By the way, it IS already on the master, it just does not work ;))

The problem is: the marquee effect only works if the element has focus (which is quiet stupid, android should always slide or at least provide the option to slide without focus)

Reply:
> (By the way, it IS already on the master, it just does not work ;))

It does work when the view is focused: What i mean is that i don't think that having multiple textviews scrolling is a good idea.

One can always check the details view to get the full title (and additional information)

Reply:
I agree with you, but i expect that this will only happen rarely, either album, title, or artist needs to exceed the width, and that is only for very rare occasions true (mostly if you have not set your tags properly)

Yeah, it only works when the view is focused, but how would you focus an element which you cant focus? :D my dev installation only scrolls if i add this custom view, otherwise it will just cut off